config/templates/cluster-init-secure.yaml

```diff
@@ -16,7 +16,14 @@ spec:
         imagePullPolicy: IfNotPresent
         volumeMounts:
         - name: client-certs
```

Comment:
Just to verify, should this name be duplicated for both items?  (same question elsewhere as well)

Reply:
ya to mount multiple volumes to the same directory you have to call out each file specifically